### PR TITLE
fix migrate.sh for windows

### LINKED ISSUE #317: Make migrate.sh windows compatible again
### Describe the bug

There was a change to the migrate.sh adding an option, causing it to fail on windows

### Describe how to produce the bug

Run the migrate sh on windows

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by DasProffi**
## Which issues does this pull request close?
closes #317

### CODE REVIEW

**Comment by FoseFx** [comment]
(on linux) you can't confirm down migrations without this

File: migrate.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ POSTGRES_URL="postgres://$POSTGRES_USER:$POSTGRES_PASSWORD@$POSTGRES_HOST:$POSTG
 
 echo "Note: using sslmode=disable, only use this script with local databases!"
 
-docker run --rm -ti -v /$(pwd)/$WORKING_DIRECTORY/migrations:/migrations --network host migrate/migrate -database $POSTGRES_URL -path migrations "$@"
+docker run --rm -v /$(pwd)/$WORKING_DIRECTORY/migrations:/migrations --network host migrate/migrate -database $POSTGRES_URL -path migrations "$@"
```
Comment (FoseFx) [inline migrate.sh:31]:
```suggestion
docker run --rm -i -v /$(pwd)/$WORKING_DIRECTORY/migrations:/migrations --network host migrate/migrate -database $POSTGRES_URL -path migrations "$@"
```

maybe this works for both windows and linux

**Comment by DasProffi** [comment]
> (on linux) you can't confirm down migrations without this

Windows neither, but you can use `./migrate.sh task-svc down 1` to down grade one version

**Review by FoseFx — APPROVED** [review]

**Comment by use-to** [comment]
Windows sucks.

**Comment by florianjs** [comment]
> Windows sucks.

I suggest to use Linux instead

**Pull request merged**